src/api_routes.rs

```diff
@@ -183,7 +183,7 @@ pub fn config(cfg: &mut ServiceConfig, rate_limit: &RateLimitCell) {
       // Site
       .service(
         scope("/site")
-          .route("", get().to(get_site_v4))
+          .route("", get().to(get_site))
```

Comment:
On `L181` I still kept this under `api/v4`, but I'm not sure we really need it. Seems like apps / clients will have to use version in nodeinfo or coming back from `GetSite` anyway, to decide which API version to use.

Reply:
> I'm not sure we really need it

What exactly do you mean?

Reply:
Instead of serving these under `/api/v4`, we could just serve them under `/api`

Reply:
People might create simple scripts that use the api without checking nodeinfo

Reply:
If we want to make any breaking changes in the future it will be much easier with a versioned api.